src/Storages/MergeTree/ReplicatedMergeTreeTableMetadata.cpp

```diff
@@ -155,8 +159,18 @@ void ReplicatedMergeTreeTableMetadata::read(ReadBuffer & in)
     else
         index_granularity_bytes = 0;
 
-    if (checkString("constraints: ", in))
-        in >> constraints >> "\n";
+    String verb;
```

Comment:
what does `verb` mean here?

Reply:
Maybe 'action' would be a better choice...